amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4018,6 +4130,91 @@ definitions:
     properties:
       reason:
         type: string
+  SearchRequestQuery:
+    type: object
+    required:
+      - query
+    properties:
+      language:
+        description: The query language in which the query is written.
+        type: string
+        default: afts
+        enum:
+          - afts
+          - lucene
+          - cmis
+      userQuery:
+        description: The exact search request typed in by the user
```

Comment:
I don't think you need the word "exact" here. To me that suggests you're referring to "exact term search" - https://docs.alfresco.com/search-enterprise/latest/config/#exact-term-search

Reply:
Removed. I was referring to the following description: https://github.com/Alfresco/rest-api-explorer/blob/master/src/main/webapp/definitions/alfresco-search.yaml#L597